src/pages/Home.jsx

```diff
@@ -52,8 +52,8 @@ const Home = () => {
               isReallySmallScreen={isReallySmallScreen}
               componentImageSrc="/assets/app-green.png"
               componentImageAlt=""
-              title="An App Built for Caseworkers"
```

Comment:
I'm not sure the spelling changes on this page are necessary.

Reply:
The spelling changes make things easier to adjust the different lines when making the website responsive.

For example, when this was changed to mobile, if we were to keep the spelling, it'll become three lines with "for" hanging in the middle.

Reply:
It seems like caseworker is the common/formal spelling, I think "An app for caseworkers" is sufficient and puts it at two lines.  its implied the app was built because it exists. With the same dimensions you are viewing its 3 lines with or without the space.  🤷‍♂️. 

I would like to re-tune this home page in the near future anyways as its somewhat verbose and I don't think our priority landing statement needs to be about decentralized technology, more so of what the app actually does. Either way thats a discussion for a work session or meeting.

Reply:
Think that should be fine. I could have this trimmed for now to "An app for caseworkers" (the phrasing works out). With regards to wording and things like that, I'll leave it for that discussion.

Reply:
I guess another edge case would be the title for the Key Features section below. For the time being I have it split up:

![Screen Shot 2023-10-24 at 12 15 31](https://github.com/codeforpdx/PASS/assets/14917816/bc241008-d7ba-41a8-a5ea-d091bfb66103)

Without it, it'll look like this:

![Screen Shot 2023-10-24 at 12 15 41](https://github.com/codeforpdx/PASS/assets/14917816/73e4198e-cf86-455d-a692-6d65a51671e6)

So a rephrasing of this might be good too. (At least for the time being.) I was thinking along the lines of "Nonprofit-Caseworker Integration" as a in-between of the two, but it can be adjusted:

![Screen Shot 2023-10-24 at 12 19 22](https://github.com/codeforpdx/PASS/assets/14917816/07dadad1-49af-4da9-8c1d-03fe4cb7930e)